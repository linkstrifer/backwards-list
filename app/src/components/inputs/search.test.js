import React from 'react';
import renderer from 'react-test-renderer';

import SearchComponent from "./search.component";

test('Search snapshot', () => {
  const component = renderer.create(
    <SearchComponent />
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})