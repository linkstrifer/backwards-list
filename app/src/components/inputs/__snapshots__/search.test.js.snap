// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot 1`] = `
<input
  onInput={[Function]}
  placeholder="buscar"
/>
`;
